pydiscourse/client.py

```diff
@@ -1281,7 +1285,7 @@ def tag_group(self, name, tag_names, parent_tag_name=None, **kwargs):
         kwargs["parent_tag_name"] = parent_tag_name
         return self._post("/tag_groups", json=True, **kwargs)["tag_group"]
 
-    def _get(self, path, **kwargs):
+    def _get(self, path, request_kwargs={}, **kwargs):
```

Comment:
One small but critical change to these HTTP utility functions: the default value of an argument should not be a mutable value, like a dict. Absent replacing this with an immutable dict (which would require another package) these should all be replaced with `request_kwargs=None` and then inside the method ensure this is a dictionary, e.g. `request_kwargs = request_kwargs or {}`.

Reply:
They were actually bugging me also, but I wanted to match the existing style.

I went and cleaned them up in the other utility methods.